README.md

```diff
@@ -519,225 +385,416 @@ project if it isn't already present in the `Packages/manifest.json` file.
-   1. Redistribute EDM4U Unity plugin with your plugin.
-      See the [Plugin Redistribution](#plugin-redistribution) for the details.
+1.  Add the `gvh` asset label to each asset (file) you want Version Handler to
+    manage.
+
+1.  Add the `gvh_version-VERSION` label to each asset where `VERSION` is the
+    version of the plugin you're releasing (e.g 1.2.3).
+
+1.  Add the `gvhp_exportpath-PATH` label to each asset where `PATH` is the
+    export path of the file when the `.unitypackage` is created. This is used to
+    track files if they're moved around in a project by developers.
+
+1.  Optional: Add `gvh_targets-editor` label to each editor DLL in your plugin
+    and disable `editor` as a target platform for the DLL. The Version Handler
+    will enable the most recent version of this DLL when the plugin is imported.
+
+1.  Optional: If your plugin is included in other Unity plugins, you should add
+    the version number to each filename and change the GUID of each asset. This
+    allows multiple versions of your plugin to be imported into a Unity project,
+    with the Version Handler component activating only the most recent version.
+
+1.  Create a manifest text file named `MY_UNIQUE_PLUGIN_NAME_VERSION.txt` that
+    lists all the files in your plugin relative to the project root. Then add
+    the `gvh_manifest` label to the asset to indicate this file is a plugin
+    manifest.
+
+1.  Optional: Add a `gvhp_manifestname-NAME` label to your manifest file to
+    provide a human readable name for your package. If this isn't provided the
+    name of the manifest file will be used as the package name. NAME can match
+    the pattern `[0-9]+[a-zA-Z -]` where a leading integer will set the priority
+    of the name where `0` is the highest priority and preferably used as the
+    display name. The lowest value (i.e highest priority name) will be used as
+    the display name and all other specified names will be aliases of the
+    display name. Aliases can refer to previous names of the package allowing
+    renaming across published versions.
+
+1.  Redistribute EDM4U Unity plugin with your plugin. See the
+    [Plugin Redistribution](#plugin-redistribution) section for details.
 
 If you follow these steps:
 
-   * When users import a newer version of your plugin, files referenced by the
-     older version's manifest are cleaned up.
-   * The latest version of the plugin will be selected when users import
-     multiple packages that include your plugin, assuming the steps in
-     [Plugin Redistribution](#plugin-redistribution) are followed.
+*   When users import a newer version of your plugin, files referenced by the
+    older version's manifest are cleaned up.
+
+*   The latest version of the plugin will be selected when users import multiple
+    packages that include your plugin, assuming the steps in
+    [Plugin Redistribution](#plugin-redistribution) are followed.
+
+## Background
```

Comment:
Why move the Background section?  I feel it makes more sense at the top, since it gives a high-level overview of what EDM4U solves.